en/docs/install-and-setup/setup/setting-up-databases/changing-default-databases/changing-to-mysql.md

```diff
@@ -67,6 +67,17 @@ Follow the  instructions below to set up a MySQL database:
         - If you are using MySQL to configure your datasource, we recommend that you use a case sensitive database collation. For more information, see the [MySQL Official Manual](https://dev.mysql.com/doc/refman/5.7/en/charset-mysql.html). The default database collation, which is `latin1_swedish_ci`, is case insensitive. However, you need to maintain case sensitivity for database collation, because when the database or table has a case-insensitive collation in MySQL 5.6 or 5.7, if a user creates an API with letters using mixed case, deletes the API, and then creates another API with the same name, but in lower case letters, then the later created API loses its permission information because when deleting the API, it keeps the Registry collection left behind.
         
         - This issue could be avoided if you use a case sensitive collation for database and tables. In that case, when creating the second API (which has the same name, but is entirely in lowercase letters), it will create a new record with the lowercase name in the `UM_PERMISSION` table.
+
+        - **UTF8/UTF8MB4 charset support**: While WSO2 recommends using `latin1` character set for MySQL databases due to product-level limitations, customers who require support for Chinese or non-ASCII characters may need to use UTF8 or UTF8MB4 character sets. When using UTF8/UTF8MB4 character sets, you must alter certain UUID columns to reduce their length to 64 characters to avoid index key length issues. Execute the following ALTER commands after creating the database tables:
+
```

Comment:
@wso2-engineering-bot These are the tables and columns that need to be reduced in length.

CREATE TABLE IF NOT EXISTS AM_API_ENDPOINTS (
    API_UUID VARCHAR(256) NOT NULL,
    ENDPOINT_UUID VARCHAR(256) NOT NULL,
    REVISION_UUID VARCHAR(255) NOT NULL DEFAULT 'Current API',
    ENDPOINT_NAME VARCHAR(255),
    KEY_TYPE VARCHAR(100),
    ENDPOINT_CONFIG LONGBLOB NOT NULL,
    ORGANIZATION VARCHAR(100),
    FOREIGN KEY (API_UUID) REFERENCES AM_API(API_UUID) ON DELETE CASCADE,
    PRIMARY KEY (API_UUID, ENDPOINT_UUID, ENDPOINT_NAME, REVISION_UUID, ORGANIZATION)
)ENGINE INNODB;

Columns that need to be reduced in length: API_UUID, ENDPOINT_UUID, REVISION_UUID